docs/Horizon/Horizon-overview.md

```diff
@@ -0,0 +1,73 @@
+# Horizon RWA Instance
+
+## Context
+- Horizon is an initiative by Aave Labs focused on creating Real-World Asset (RWA) products tailored for institutions.
+- Horizon will launch a licensed, separate instance of the Aave Protocol (initially a fork of v3.3) to accommodate regulatory compliance requirements associated with RWA products.
+- Horizon Instance will have a dual role setup with responsibilities split between Aave DAO (Operational role) and Aave Labs (Executive role).
+ 
+## Overview
+The Horizon Instance will introduce permissioned (RWA) assets. The Aave Pool will remain the same, except that RWA assets can be used as collateral in order to borrow stablecoins. Permissioning occurs at the asset level, with each RWA token issuer enforcing asset-specific restrictions directly into their ERC-20 token. The Aave Pool is agnostic to each specific RWA implementation and its asset-level permissioning.
+
+From an Issuer perspective, aTokens are an extension of the RWA tokens, which are securities. The aTokens will signify ownership of the supplied underlying RWA Token. To accomodate edge cases, a protocol-wide aToken Transfer Admin is also added, allowing Issuers the ability to forcibly transfer aTokens on behalf of end users (without needing approval). These transfers will still enforce collateralization and health factor requirements as in existing Aave peer-to-peer aToken transfers.
+
+There is permissionless supply of USDC to the Horizon Instance which can earn yield. GHO will also be listed in Horizon as a standard, non-mintable stablecoin. Borrowable stablecoins will not be used as collateral. 
```

Comment:
why do we mention USDC here specifically?

Reply:
In this last paragraph I’d speak more general about the pool: i) any asset can be listed through the usual process, like a standard Aave instance, ii) this instance is primarily aimed at onboarding stablecoins for borrowing, such as USDC and GHO among others